src/sdf_tools/utils.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import sdf_tools
+
+
+def compute_2d_sdf_and_gradient(grid_world, sdf_resolution, sdf_origin, frame='/world'):
+    """
+    :param grid_world: a 2d numpy array of the world full of 0 and 1
+    :param sdf_resolution: float in meters
+    :param sdf_origin: size 2 np.array (vector) in meters
+    :param frame: unused
```

Comment:
This might be *slightly* misleading, but not a lot. `frame` gets passed to `CollisionMapGrid` but is never read by anything `downstream` before being deconstructed.

Reply:
if it's never read I call that unused